ext/mysqli/tests/bug73462.phpt

```diff
@@ -5,6 +5,13 @@ mysqli
 --SKIPIF--
 <?php
 require_once('skipifconnectfailure.inc');
+/*
+ * XXX: this test is flaky with persistent connections on PPC CI runner
```

Comment:
What's the xxx for?

Reply:
TODO marker for things that are broken specifically